src/index.js

```diff
@@ -267,120 +265,144 @@ async function establishBridging(hubState, bridges) {
 
   const lastPresenceMessages = {}; // { discordCh: message object }
   presenceRollups.on('new', ({ kind, users, fresh }) => {
-    if (statsdClient != null) {
-      statsdClient.send("reticulum.presencechanges", 1, "c");
-    }
-    for (const discordCh of bridges.getChannels(hubState.id).values()) {
-      if (VERBOSE) {
-        console.debug(ts(`Relaying presence ${kind} in ${hubState.id} to ${formatDiscordCh(discordCh)}.`));
-      }
-      if (kind === "arrive") {
-        const msg = fresh ?
-          `${formatList(users)} joined the Hubs room. Join them: ${hubState.shortUrl}` :
-          `${formatList(users)} joined the Hubs room.`;
-        lastPresenceMessages[discordCh.id] = discordCh.send(msg);
-      } else if (kind === "depart") {
-        const msg = `${formatList(users)} left the Hubs room.`;
-        lastPresenceMessages[discordCh.id] = discordCh.send(msg);
-      } else if (kind === "rename") {
-        lastPresenceMessages[discordCh.id] = discordCh.send(formatRename(users[0]));
+    try {
+      if (statsdClient != null) {
+        statsdClient.send("reticulum.presencechanges", 1, "c");
       }
+      for (const discordCh of bridges.getChannels(hubState.id).values()) {
+        if (VERBOSE) {
+          console.debug(ts(`Relaying presence ${kind} in ${hubState.id} to ${formatDiscordCh(discordCh)}.`));
+        }
+        if (kind === "arrive") {
+          const msg = fresh ?
+            `${formatList(users)} joined the Hubs room. Join them: ${hubState.shortUrl}` :
+            `${formatList(users)} joined the Hubs room.`;
+          lastPresenceMessages[discordCh.id] = discordCh.send(msg);
+        } else if (kind === "depart") {
+          const msg = `${formatList(users)} left the Hubs room.`;
+          lastPresenceMessages[discordCh.id] = discordCh.send(msg);
+        } else if (kind === "rename") {
+          lastPresenceMessages[discordCh.id] = discordCh.send(formatRename(users[0]));
+        }
+      }
+    } catch (e) {
+      console.error(ts(`Error relaying presence ${kind} in ${hubState.id}.`));
```

Comment:
Should all these `catch` logs also include the error object, or at least `e.name` or `e.message`?